### PR TITLE
2870: POI filters: Button to only show currently open locations can be cut off

### PR DESCRIPTION
**Pull request by bahaaTuffaha**
### Short description

The toggle button is cut off due to long sentences from languages like French.

### Proposed changes

- Added a max-width to StyledText to leave some space to the toggle button.
- Added `alignItems: 'center'` to Row style just to make the text,icon and toggle aliened vertically on the same level. 

### Side effects

None

### Resolved issues

Fixes: #2870 

---

### CODE REVIEW

File: release-notes/unreleased/2870-POI-filters-fix-toggle-button.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+issue_key: 2870
+show_in_stores: false
+platforms: # relevant platforms, possible values: web, android and ios
+  - android
+  - ios
+en: A Fix for the toggle button at the poi filter.
```
Comment (LeandraH) [inline release-notes/unreleased/2870-POI-filters-fix-toggle-button.yml:6]:
```suggestion
en: A fix for the toggle button in the map filter
de: Der Toggle-Button im Kartenfilter wurde verbessert
```
We call them pois internally but most users probably don't know that that means point of interest.

**Review by LeandraH — COMMENTED** [review]
Thanks for the PR! It looks pretty good already but I think the button should ideally be in the same line as the icons under it. Maybe you can do something with flex positioning instead of a hard-coded width?

IMG_255E967AC9BF-1

**Review by LeandraH — APPROVED** [review]
Nicely done, thank you! Tested in real iOS

**Review by steffenkleinle — APPROVED** [review]
Tested on android, nice solution! :)

**Comment by LeandraH** [comment]
By the way, you can enable auto-merge so that you don't have to come back to the PR to merge it when the pipeline is finished :)

**Pull request merged**